### PR TITLE
Upgrade to ES 7.6.2 and release version 7.6.2.0

### PR DESCRIPTION
**Pull request by vvanholl**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -105,14 +105,6 @@ esplugin {
...
 }
 
-// Remove in 7.6.2
-// https://github.com/spinscale/elasticsearch-ingest-langdetect/blob/c619173300113f26411a0bc42da8e41c8d4d72a5/build.gradle#L26
-restResources {
-    restApi {
-        includeCore '*'
-    }
-}
-
```
Comment (lukas-vlcek) [inline build.gradle:115]: I just check with the origin (the URL in comment) and it seems they did not remove it for 7.6.2 release, see https://github.com/spinscale/elasticsearch-ingest-langdetect/commit/8066ad5f4427a361ee21054b3658af9034ef43ec

Comment (lukas-vlcek) [reply build.gradle:115]: May be we should it keep for the time being? I usually run into issues with this if I run repeated tests locally and gradle is using local cache of the dependencies. Removing the cache (i.e. `rm -rf ~/.gradle`) helps which indicates that if you run the test on a fresh machine then you do not run into issues.

Comment (vvanholl) [reply build.gradle:115]: OK let's keep it.

**Comment by ArtyomGabeev** [comment]
@lukas-vlcek any plans to get it merged and released?

**Comment by lukas-vlcek** [comment]
@ArtyomGabeev sorry for delay. I will get to this by the end of this week.

**Pull request merged**